hoomd/device.py

```diff
@@ -67,264 +54,320 @@ def __init__(self, communicator, notice_level, msg_file, shared_msg_file):
 
     @property
     def comm(self):
-        """
-        Get the MPI Communicator
-        """
-
+        warnings.warn("Use communicator.", DeprecationWarning)
```

Comment:
Is the plan to remove this for the 3.0 final release? If so we may want to mark that somehow. I know that the `deprecated` package helps with that, but We don't have enough major releases to warrant that likely.

Reply:
Yes, we will remove this before the release #755 . For now, `device.mode` is used heavily throughout the codebase.

Reply:
nvm, will just make the change now and other PRs will need to adapt before merging.